### DISCUSSION TITLE
Where I can see available config for this package?

### DISCUSSION THREAD

**Discussion by lutfifirdaus** [saade/filament-fullcalendar#206] [Q&A]
So I wanna set custom height and I try to put it on config but isn't working and I try other setting from fullcalendar JS site but there is some not working too. Help pls

**Comment by wbpols** [comment]
Did you already try the following in your `\App\Providers\Filament\PanelProvider`:
```
\Saade\FilamentFullCalendar\FilamentFullCalendarPlugin::make()
    ->config([
        'height' => 200,
    ]),
```
